colcon_core/verb/build.py

```diff
@@ -133,6 +160,45 @@ def main(self, *, context):  # noqa: D102
         jobs, unselected_packages = self._get_jobs(
             context.args, decorators, install_base)
 
+        underlay_packages = {}
+        for prefix_path in get_chained_prefix_path():
+            packages = find_installed_packages(Path(prefix_path))
+            for pkg, path in packages.items():
```

Comment:
We might want to guard against `packages` being `None` (it's a possible return value).

Reply:
Guards against packages being `None` in ac6cf92